### ISSUE TITLE
[TF2] Place where you can get stuck and shoot blue team in pl_upward

### ISSUE THREAD

**Issue by thejaviertc** [ValveSoftware/Source-1-Games#4104]
There is a place where you can rocketjump/stickyjump, you get stuck and you can shoot the blue team.

It's a bit weird because sometimes it works and sometimes it doesn't.

https://user-images.githubusercontent.com/45662206/180752697-0a9c70f9-7dac-4ad0-9817-071edf81a5bf.mp4

https://user-images.githubusercontent.com/45662206/180752688-833fedb8-fbbe-4f9f-866a-03be393d403d.mp4

It can be abused as [this video that I found on Reddit](https://www.reddit.com/r/tf2/comments/w77tcz/i_discovered_that_you_can_use_a_wallbug_to_make/?utm_source=share&utm_medium=web2x&context=3) demonstrates it.

**Comment by Mooselk** [comment]
Weird to see the general player base discover wall bugs after 15 years.

**Comment by RicochetYT** [comment]
Yeah, this is due to the collision box sticking to weird corners. This happens on any map where the wall is angled, to my knowledge?

**Comment by ktaeoh** [comment]
casual players when they discover wallbugs
